### DISCUSSION TITLE
TypeError: props.render is not a function

### DISCUSSION THREAD

**Discussion by TOcvfan** [react-hook-form/react-hook-form#7371] [Q&A] [answered]
I keep getting this error, I have tried to reinstall react-hook-form and I have checked stackoverflow and I found an error but that wasn't it (as={} instead of render), but I keep gettind the samme error

```
<Controller
        render={({ field }) =>
          <TextField
            {...field}
          />
        }
        control={control}
        name={name}
        variant="outlined"
        InputLabelProps={{
          shrink: true,
        }}
      />
```

**Comment by bluebill1049** [comment]
can you reproduce this issue in a codesandbox?

**Comment by TOcvfan** [comment]
No I can't reproduce the error in sandbox
[https://codesandbox.io/s/patient-dawn-eovwh?file=/src/App.js](url)
it works in sandbox but not on my computer I'm lost...

**Reply by bluebill1049** [reply]
are you using the same version between yours and the codesandbox?

**Reply by TOcvfan** [reply]
Yes I took the time to make sure it was the same version

**Comment by TOcvfan** [comment, marked as answer]
I tried to make a new react app but it stalls, so its my computer that is the problem
![Skærmbillede 2021-12-29 170218](https://user-images.githubusercontent.com/11290288/147681234-c8b42a6a-2c95-481a-b3bf-18cd43ab6cb3.png)
